docs/proposals/kubeedgeadm-scope.md

```diff
@@ -261,3 +261,4 @@ Flags:
 
     1. Remove node using `kubectl` command
     2. Kill `edge_core` process
+  
```

Comment:
Can we remove this extra blank line ?

Reply:
This file is not part of review. The update of this document is part of PR#489 , which is already merge.